src/main/java/net/dv8tion/jda/api/events/message/GenericMessageEvent.java

```diff
@@ -26,6 +26,16 @@
  * <br>Every MessageEvent is an instance of this event and can be casted.
  * 
  * <p>Can be used to detect any MessageEvent.
+ *
+ * <h2>Requirements</h2>
+ *
+ * <p>This event requires at least one of the following intents:
+ * <ul>
+ *     <li>{@link net.dv8tion.jda.api.requests.GatewayIntent#GUILD_MESSAGES GUILD_MESSAGES} to work in guild text channels
+ *     <li>{@link net.dv8tion.jda.api.requests.GatewayIntent#DIRECT_MESSAGES DIRECT_MESSAGES} to work in private channels
```

Comment:
For better compatibility, we also include the `</li>` end tags. HTML/5 is not required to build javadocs.